test/demo/utils/common.ts

```diff
@@ -0,0 +1,110 @@
+import {Identity} from 'openfin/_v2/main';
+
+import {Intent} from '../../../src/client/intents';
+import {testManagerIdentity} from '../constants';
+
+import {fin} from './fin';
+import * as fdc3Remote from './fdc3RemoteExecution';
+import {delay} from './delay';
+
+export type Boxed<T> = {value: T}
+
+/**
+ * Identity with a mandatory `name`
+ */
+export interface AppIdentity extends Identity {
+    name: string; // Make inherited field mandatory
+}
+
+export interface DirectoryAppIdentity extends AppIdentity {
+    appId: string;
+}
+
+export interface NonDirectoryAppIdentity extends AppIdentity {
+    manifestUrl: string;
+}
+
+/**
+ * Quits an OpenFin app (or multiple in parallel) using `force=true` and swallowing errors
+ * @param apps Identities of the apps to quit
+ */
+export async function quitApp(...apps: Identity[]) {
```

Comment:
Annoying and nitpicky, but it would make more sense to me if this was called `quitApps` since it takes multiple.